### ISSUE TITLE
Acces document's field without using .get/.set

### ISSUE THREAD

**Issue by LennyPenny** [vadimdemedes/mongorito#200]
Is there a plan for this so documents could be used more natively? 

i.e. 
```js
let user = await User.findOne();

user.name = user.name + " test";

user.save()
```

I'm not familiar with your current implementation, but it could probably easily be implimented using [proxies](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)

**Comment by vadimdemedes** [comment]
Yes, I think it's ok to implement this now since proxies have good Node.js support now.

**Comment by LennyPenny** [comment]
Sadly I'm having trouble implementing this because I'm really familiar with the underlying react stuffs 

![https://vgy.me/CkMBZa.png](https://vgy.me/CkMBZa.png)

Could you point me in the right direction?

**Comment by vadimdemedes** [comment]
Hm, yeah, I did a little research and I don't think it's possible to implement that feature with a Proxy. Seems there's no clean solution for it. Plus it's not worth all the effort to just save a few chars.
